Formula/burp.rb

```diff
@@ -13,19 +17,60 @@ class Burp < Formula
     sha256 "d3b8cc95839835dd9ba9574444d5cf7394d398376fe2a06fcd3ba9a00b395476" => :mavericks
   end
 
+  devel do
+    url "https://github.com/grke/burp/archive/2.0.42.tar.gz"
+    sha256 "9a698f43d9c5c09c73883f7389fcc5889d79fb4452c8478480a790df087924f8"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+
+    resource "uthash" do
+      url "https://github.com/troydhanson/uthash.git",
+        :revision => "bc910acb678a9f3aeeec1b1d9e0d7d0964a8c349"
+    end
+  end
+
+  head do
+    url "https://github.com/grke/burp.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+
+    resource "uthash" do
+      url "https://github.com/troydhanson/uthash.git"
+    end
+  end
+
   depends_on "librsync"
   depends_on "openssl"
 
-  # patches to change directories to brew conventions in Makefile and config
-  # files
-  patch :DATA
-
   def install
-    system "./configure", "--prefix=#{prefix}",
-                          "--sysconfdir=#{etc}/burp",
-                          "--sbindir=#{bin}",
-                          "--localstatedir=#{var}/burp"
-    system "make", "install"
+    args = %W[
+      --prefix=#{prefix}
+      --sysconfdir=#{etc}/burp
+      --localstatedir=#{var}
+    ]
+    if build.stable?
+      args << "--sbindir=#{sbin}"
+      system "./configure", *args
+      system "make", "install"
+    else
+      resource("uthash").stage do
+        system "make", "-C", "libut"
+        (buildpath/"uthash/lib").install "libut/libut.a"
+        (buildpath/"uthash/include").install Dir["src/*"]
+      end
+
+      ENV.prepend "CFLAGS", "-I#{buildpath}/uthash/include"
+      ENV.prepend "CXXFLAGS", ENV.cflags
```

Comment:
Hmm I wonder if this should be just CPPFLAGS instead of CFLAGS and CXXFLAGS separately since it's setting an include.